cfme/tests/configure/test_default_views_cloud.py

```diff
@@ -1,29 +1,34 @@
 # -*- coding: utf-8 -*-
-
 import pytest
+
 from cfme import test_requirements
 from cfme.cloud.provider import CloudProvider
+from cfme.configure.settings import DefaultView
 from cfme.fixtures import pytest_selenium as sel
-import cfme.web_ui.toolbar as tb
-from cfme.web_ui import ButtonGroup, form_buttons, Quadicon, fill
+from cfme.web_ui import Quadicon, fill, toolbar as tb
 from utils.appliance.implementations.ui import navigate_to
 from utils.providers import setup_a_provider as _setup_a_provider
-from cfme.configure import settings  # NOQA
-from cfme.cloud import instance  # NOQA
 
 pytestmark = [pytest.mark.tier(3),
-              test_requirements.settings]
-
+              test_requirements.settings,
+              pytest.mark.usefixtures('setup_a_provider')]
 
 gtl_params = {
     'Cloud Providers': CloudProvider,
     'Availability Zones': 'clouds_availability_zones',
     'Flavors': 'clouds_flavors',
     'Instances': 'clouds_instances',
-    'Images': 'clouds_images'
+    # clouds_images DNE, replace with proper object with navmazing destinations
+    # 'Images': 'clouds_images'
 }
 
 
+# TODO refactor for setup_provider parametrization with new 'latest' tag
```

Comment:
Earlier discussion resulted in reverting testgen parametrization of the provider. Adding this TODO for a prototype effort to provide a 'latest' tag per-provider in YAML.